docs/source/cognite_client.rst

```diff
@@ -5,12 +5,11 @@ CogniteClient
     :members:
     :member-order: bysource
 
+.. _class_client_ClientConfig:
```

Comment:
I added an explicit label as it appears to be impossible to link to autogenerated classes. The link works well in my locally built docs.